tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Autogenerated by NextJS, see: https://nextjs.org/docs/basic-features/typescript

Have left the configuration as is